src/main/java/Duke.java

```diff
@@ -1,10 +1,87 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/*
+Level-1
+idea:
+- design a "reply" method to format the duke reply text
+- print hello statement
+- scanf input
+    - print indented horizontal line
+    - print input
+    - print indented horizontal line
+
+Level-2
+idea:
+- addTask method to add to static array of strings
+- formatTaskList to take task array and output string
+    in list form
+- if list command, reply(formatTaskList(taskArray))
+ */
 public class Duke {
-    public static void main(String[] args) {
+    private TaskList taskList = new TaskList();
+
+    private void reply(String s) {
+        System.out.println("\t"
+                + "____________________________________________________________");
+        System.out.println("\t" + s.replace("\n", "\n    "));
+        System.out.println("\t"
+                + "____________________________________________________________");
+    }
+
+    private void greet() {
```

Comment:
Would you consider encapsulating all of Duke's interactions with the user into a separate class? The class could be the `Ui` class, containing the methods `greet()`, `printExitMessage()`, etc.